rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java

```diff
@@ -1436,11 +1439,11 @@ void saveLockWhiteListAfterGetWithData() {
                 .thenReturn(new byte[]{(byte) 0xee});
 
             Mockito
-                    .doAnswer((InvocationOnMock invocation) -> {
-                        storageCalled.set(Boolean.TRUE);
-                        return null;
-                    })
-                    .when(repositoryMock).addStorageBytes(any(RskAddress.class), eq(LOCK_ONE_OFF_WHITELIST_KEY.getKey()), eq(new byte[]{(byte) 0xee}));
+                .doAnswer((InvocationOnMock invocation) -> {
+                    storageCalled.set(Boolean.TRUE);
+                    return null;
+                })
+                .when(repositoryMock).addStorageBytes(any(RskAddress.class), eq(LOCK_ONE_OFF_WHITELIST_KEY.getKey()), eq(new byte[]{(byte) 0xee}));
```

Comment:
nit. If we can pass the actual values, it would be better and less error pronne, like `eq(actualRskAddress)`, if possible.

Reply:
i agree, but thats out of scope